addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -19,153 +17,242 @@
+                    data-bs-toggle="offcanvas"
+                    data-bs-target="#o_wcrm_offcanvas">
+                    <i class="fa fa-sliders"/>
+                </button>
             </div>
-            <div>
-                <p t-if="not partners">No result found</p>
-                <t t-foreach="partners" t-as="partner">
-                    <t t-if="last_grade != partner.grade_id.id">
-                        <h3 class="text-center mt-4">
-                            <span t-field="partner.grade_id"/>
-                            <t t-call="website.publish_management">
-                                <t t-set="object" t-value="partner.grade_id"/>
-                                <t t-set="publish_edit" t-value="True"/>
-                            </t>
-                        </h3>
-                        <t t-set="last_grade" t-value="partner.grade_id.id"/>
-                    </t>
-                    <div class="d-flex mt-3">
-                        <a t-attf-href="/partners/#{slug(partner)}?#{current_grade and 'grade_id=%s&amp;' % current_grade.id}#{current_country and 'country_id=%s' % current_country.id}"
-                           t-field="partner.avatar_128"
-                           class="me-3 text-center o_width_128"
-                           t-options='{"widget": "image", "qweb_img_responsive": False, "class": "o_image_128_max"}'
-                        ></a>
-                        <div class="flex-grow-1 o_partner_body" style="min-height: 64px;">
-                            <a t-attf-href="/partners/#{slug(partner)}?#{current_grade and 'grade_id=%s&amp;' % current_grade.id}#{current_country and 'country_id=%s' % current_country.id}">
-                                <span t-field="partner.display_name"/>
-                            </a>
-                            <div t-field="partner.website_short_description"/>
-                            <t t-if="any(p.website_published for p in partner.implemented_partner_ids)">
-                                <small><a t-attf-href="/partners/#{slug(partner)}#right_column">
-                                    <t t-esc="partner.implemented_partner_count"/> reference(s)
-                                </a></small>
-                            </t>
+        </div>
+        <!-- Off canvas filters on mobile-->
+        <div id="o_wcrm_offcanvas" class="o_website_offcanvas offcanvas offcanvas-end d-lg-none p-0 overflow-visible">
+            <div class="offcanvas-header">
+                <h5 class="offcanvas-title">Filters</h5>
+                <button type="button" class="btn-close" data-bs-dismiss="offcanvas" aria-label="Close"/>
+            </div>
+            <div class="offcanvas-body p-0">
+                <div class="accordion accordion-flush">
+                    <div class="accordion-item">
+                        <h2 class="accordion-header">
+                            <button class="accordion-button fw-bold collapsed"
```

Comment:
I don't think that's a good idea to **force** bold here... it may look ok with the default theme but we don't know what font the user we'll choose...